lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +79,52 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as f:
+            config = json.load(f)
+
+        return ConfigDTO(**config)
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        if not (
+            isinstance(self.config_dto.seed_urls, list)
+            and all(
+                re.match(r"https?://(www.)?securitylab\.ru/news", seed_url)
+                for seed_url in self.config_dto.seed_urls
+                    )
+                ):
+            raise IncorrectSeedURLError
+    
+        if (
+            not isinstance(self.config_dto.total_articles, int)
+            or self.config_dto.total_articles <= 0
+        ):
+            raise IncorrectNumberOfArticlesError
+
+        if (
+            self.config_dto.total_articles > 0 and
```

Comment:
strange condition. you should check that value is within the range and fail otherwise.